cirq-google/cirq_google/calibration/xeb_wrapper_test.py

```diff
@@ -0,0 +1,131 @@
+# Copyright 2021 The Cirq Developers
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import multiprocessing.pool
+
+import numpy as np
+import pandas as pd
+import scipy.optimize
+import scipy.optimize._minimize
+
+import cirq
+import cirq_google as cg
+from cirq.experiments import random_rotations_between_grid_interaction_layers_circuit
+from cirq.experiments.xeb_fitting import XEBPhasedFSimCharacterizationOptions
+from cirq_google.calibration.phased_fsim import (
+    LocalXEBPhasedFSimCalibrationOptions,
+    LocalXEBPhasedFSimCalibrationRequest,
+)
+from cirq_google.calibration.xeb_wrapper import (
+    run_local_xeb_calibration,
+    _maybe_multiprocessing_pool,
+)
+
+SQRT_ISWAP = cirq.ISWAP ** -0.5
+
+
+def minimize_patch(
```

Comment:
make private?

Reply:
done. didn't think it mattered much in a test file